core/src/main/java/org/frankframework/senders/ParallelSenders.java

```diff
@@ -68,21 +67,18 @@ public void configure() throws ConfigurationException {
 	}
 
 	@Override
-	public SenderResult doSendMessage(Message message, PipeLineSession session) throws SenderException, TimeoutException {
-		Guard guard = new Guard();
-		Map<ISender, ParallelSenderExecutor> executorMap = new LinkedHashMap<>();
-		boolean success = true;
-		String errorMessage = null;
-
+	public SenderResult doSendMessage(Message message, PipeLineSession session) throws SenderException {
 		try {
 			if (!message.isRepeatable()) {
 				message.preserve();
 			}
 		} catch (IOException e) {
 			throw new SenderException(getLogPrefix() + " could not preserve input message", e);
 		}
+
+		Map<ISender, ParallelSenderExecutor> executorMap = new LinkedHashMap<>();
+		CountDownLatch senderCountDownLatch = new CountDownLatch(((List<ISender>) getSenders()).size());
```

Comment:
Beter toch de `Phaser` te gebruiken hier omdat ik denk dat het er niet beter van wordt ze door elkaar te gebruiken in de `ParallelSenderExecutor`.

Reply:
Aangepast.